### ISSUE TITLE
[BUG]Issue with connecting after reinstalling program

### ISSUE THREAD

**Issue by jsl8124** [Chia-Network/chia-blockchain#6650]
**Describe the bug**
Issue connecting after reinstalling Chia.exe
System thinks it is still connected and giving an error, "Already connected, not reconnecting" 
Verbose - [Violation] 'animationiteration' handler took 549065ms

**Desktop (please complete the following information):**

- OS: [Windows]
- OS Version/Flavor: [Windows 10]
- CPU: [Intel Core i5-8300H CPU @2.30GHz]

**Additional context**
![Image 1](https://user-images.githubusercontent.com/85648894/121431681-b61ac200-c947-11eb-8bbd-768637ca1793.JPG)
[Electron js2c rederer_initjs.txt](https://github.com/Chia-Network/chia-blockchain/files/6626962/Electron.js2c.rederer_initjs.txt)

**Comment by jsl8124** [comment]
Solve this issue by...
1) In Task Manager and end tasks for all Chia programs and then delete these two files from your folders.
2) C:\Users\USERNAME.chia\mainnet\db - delete file - blockchain_v1_mainnet.sqlite
3) C:\Users\USERNAME.chia\mainnet\wallet\db - delete file - blockchain_wallet_v1_mainnet_NUMBER.sqlite

**Comment by RobbieE-ZA** [comment]
![image](https://user-images.githubusercontent.com/85682538/121518535-e10d2080-c9f0-11eb-9706-3350ebcb63f6.png)

I've been having a similar problem. I was running 1.1.3, and upgraded to 1.1.7 with no improvement.

I've tried deleting the above-mentioned database files, but that had no effect. I suspect this is not the cause of my issue.

I noticed that the browser console is constantly spitting out error messages about trying to connect to a loopback IP on port 55400. Force-reloading results in the same display as the OP. If I end the task, and start up again (even after restarting my computer), I get the error again about not being able to connect to port 55400.

I don't have anything listening on that port, and I tried editing the config file to change this port(I found 2 items listing that port). However, Chia ignores any changes I made in the config file, and insists on still trying to connect on port 55400.
